pkg/controller/serviceaccount/tokens_controller.go

```diff
@@ -42,6 +42,8 @@ import (
 	"k8s.io/kubernetes/pkg/serviceaccount"
 )
 
+const ServiceServingCASecretKey = "service-ca.crt"
```

Comment:
can't skip this carry? Didn't we remove the CA from the service account?

Reply:
We can't remove anytime soon.

Reply:
I've also dropped the commit that disabled the addition of service CA.